gemini/sample-apps/genwealth/deployment/deploy-search.sh

```diff
@@ -40,6 +40,9 @@ DATA_STORE_ID=$(curl -X GET \
 DATA_STORE_ID=${DATA_STORE_ID##*/}
 
 # Upload samples to gcs
+gsutil -m cp -r \
+  "gs://github-repo/generative-ai/sample-apps/genwealth/sample-prospectus" \
+  .
 gsutil cp sample-prospectus/*.pdf gs://"${DOCS_BUCKET}"
```

Comment:
You don't need to download these files locally, they can be transferred directly between buckets.

```
gsutil -m cp gs://github-repo/generative-ai/sample-apps/genwealth/sample-prospectus/*.pdf gs://"${DOCS_BUCKET}"
```

Reply:
Good call - fixed this.